### PR TITLE
Added translations for modal texts

### PR DESCRIPTION
**Pull request by leszczuu**

### CODE REVIEW

File: src/Resources/assets/shop/js/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import './handleEditWishlistModal';
 import { WishlistVariantButton } from './WishlistVariantButton';
 import { WishlistVariantPrice } from './WishlistVariantPrice';
 
+
```
Comment (diimpp) [inline src/Resources/assets/shop/js/index.js:9]: Unnecessary extra empty line

File: src/Resources/assets/shop/js/wishlistModal.js [hunk 2]
```diff
@@ -90,6 +91,15 @@ export class WishlistModal {
             this.actions.performAction();
             this._closeModal();
         });
+
+        if (input) {
+            input.addEventListener("keypress", function(event) {
```
Comment (diimpp) [inline src/Resources/assets/shop/js/wishlistModal.js:96]:
```suggestion
            input.addEventListener('keypress', function(event) {
```

File: src/Resources/assets/shop/js/wishlistModal.js [hunk 3]
```diff
@@ -90,6 +91,15 @@ export class WishlistModal {
...
+                if (event.key === "Enter") {
```
Comment (diimpp) [inline src/Resources/assets/shop/js/wishlistModal.js:97]:
```suggestion
                if (event.key === 'Enter') {
```

File: src/Resources/translations/messages.en.yml [hunk 4]
```diff
@@ -47,3 +47,19 @@ bitbag_sylius_wishlist_plugin:
         wishlist_saved: Wishlist has been saved.
         you_have_no_access_to_that_wishlist: You have no access to that wishlist.
         wishlist_has_product_variant: '%productName% variant is already in wishlist.'
+        wishlist_modal_copy_title: 'Choose a wishlist to which selected items should be copied'
+        wishlist_modal_copy_cancel: 'cancel'
```
Comment (diimpp) [inline src/Resources/translations/messages.en.yml:51]: The point of `bitbag_sylius_wishlist_plugin.ui` namespace is to re-use translations without creating new individual translation for each element.

These should be merged
> wishlist_modal_copy_cancel: 'cancel'
> wishlist_modal_add_cancel: 'cancel'
> wishlist_modal_edit_cancel: 'cancel'
> wishlist_modal_remove_cancel: 'cancel'

or even better, replace it with `sylius.ui.cancel`

File: features/assigning_wishlist_to_user.feature [hunk 5]
```diff
@@ -42,18 +42,18 @@ Feature: Assigning a wishlist to a user
...
-    And I log in as "jdeer@sylius.pl"
-    And I go to "/wishlists"
-    And I press "wishlist-edit-button-Wishlist"
-    And I fill in "edit_wishlist_name" with "Wishlist-not-assigned"
-    And I press "edit_wishlist_save"
-    And I log out
-    And I go to "/wishlists"
-    Then I should have 1 wishlists
-    And I should see "Wishlist-not-assigned"
+#  @ui @javascript
```
Comment (diimpp) [inline features/assigning_wishlist_to_user.feature:45]: Should it be removed entirely? What's the point of leaving dead code

File: features/assigning_wishlist_to_user.feature [hunk 6]
```diff
@@ -42,18 +42,18 @@ Feature: Assigning a wishlist to a user
...
+#    When I go to "/"
+#    And I log in as "jdeer@sylius.pl"
+#    And I go to "/wishlists"
+#    And I press "wishlist-edit-button-Wishlist"
+#    And I fill in "edit_wishlist_name" with "Wishlist-not-assigned"
+#    And I press "edit_wishlist_save"
+#    And I log out
+#    And I go to "/wishlists"
+#    Then I should have 1 wishlists
+#    And I should see "Wishlist-not-assigned"
```
Comment (senghe) [inline features/assigning_wishlist_to_user.feature:59]:
```suggestion
```

File: features/assigning_wishlist_to_user.feature (hunk 5 above)
Comment (senghe) [reply features/assigning_wishlist_to_user.feature]: Removed

File: src/Resources/translations/messages.en.yml (hunk 4 above)
Comment (senghe) [reply src/Resources/translations/messages.en.yml:51]: This is another intention. It's not overridding the translations by setting new "copy" key. Here we've translations for the "copy" functionality. It's not same with `sylius.ui.cancel` context I think.

File: src/Resources/assets/shop/js/index.js (hunk 1 above)
Comment (senghe) [reply src/Resources/assets/shop/js/index.js:9]: I think this is not relevant issue

File: src/Resources/assets/shop/js/wishlistModal.js (hunk 3 above)
Comment (senghe) [reply src/Resources/assets/shop/js/wishlistModal.js:97]: I think this is not relevant issue

**Review by senghe — APPROVED** [review]

**Pull request merged**